includes/class-wc-siftscience-admin.php

```diff
@@ -20,9 +20,14 @@
 	 * Class WC_SiftScience_Admin
 	 */
 	class WC_SiftScience_Admin {
-		private const ADMIN_ID    = 'siftsci';
-		private const ADMIN_LABEL = 'Sift';
-		private const NONCE_HOOK  = 'woocommerce_settings_siftsci';
+		private const ADMIN_ID            = 'siftsci';
+		private const ADMIN_LABEL         = 'Sift';
+		private const GET_VAR_CLEAR_STATS = 'clear_stats';
+		private const GET_VAR_SET_STATS   = 'set_siftsci_stats';
+		private const GET_VAR_RESET_GUID  = 'reset_guid';
+		private const GET_VAR_TEST_SSL    = 'test_ssl';
+		private const GET_VAR_CLEAR_LOGS  = 'clear_logs';
+		private const NONCE_ACTION_PREFIX = 'woocommerce_settings_nonce_';
```

Comment:
do you imagine how long the HTML attrib would be

Reply:
I think this is an internal string. It doesn't appear in an HTML attribute. But I can also make it shorted.